src/web/web.py

```diff
@@ -28,10 +28,15 @@
 from flask import Flask
 from flask import render_template
 
+# Flask extensions
+from flaskext.mail import Mail
```

Comment:
web_1           | -------------------------------------------------
web_1           | Traceback (most recent call last):
web_1           |   File "/code/web.py", line 32, in <module>
web_1           |     from flaskext.mail import Mail
web_1           | ImportError: No module named flaskext.mail

Fixed it in my local instance.